assets/scss/components/_dropdown.scss

```
@@ -0,0 +1,58 @@
+$dropdown-offset-right: $length-6;
+
+details.dropdown {
+    position: relative;
+
+    summary {
+        appearance: none;
+    }
+
+    &[open] {
+        z-index: 10;
+    }
+
+    &[open] summary::before {
+        content: '';
+        position: fixed;
+        top: 0;
+        right: 0;
+        bottom: 0;
+        left: 0;
+    }
+
+    .dropdown-content {
+        position: absolute;
+        top: $length-32;
+        right: $dropdown-offset-right;
+
+        margin: 0;
+        padding: 0;
+
+        z-index: 21;
+
+        @include shadow-4;
+
+        &::before,
+        &::after {
+            content: " ";
+            position: absolute;
+            display: block;
+            width: 0;
+            height: 0;
+        }
+        &::before {
+            top: -10px;
+            right: #{$length-14 - $dropdown-offset-right};
+            border: 10px solid transparent;
+            border-top: 0;
+            border-bottom-color: rgba($true-black, .1);
+        }
+        &::after {
+            top: -9px;
+            right: #{$length-15 - $dropdown-offset-right};
```

Si le but de `$length-15` était de pouvoir sommer ici, il plus propre d'utiliser l'offset comme il est, et d'y ajouter le décalage nécessaire à la construction de la flèche en pixels, à l'aide de `calc` : 

```suggestion
            right: calc(15px - #{$dropdown-offset-right});
```